libraries/SITL/SIM_AirSim.cpp

```diff
@@ -61,9 +70,9 @@ void AirSim::set_interface_ports(const char* address, const int port_in, const i
 /*
 	Decode and send servos
 */
-void AirSim::send_servos(const struct sitl_input &input)
+void AirSim::output_copter(const struct sitl_input &input)
 {
-	servo_packet pkt{0};
+    servo_packet pkt;
```

Comment:
I expect you've satisfied yourself this is pointless (and perhaps bad) as we definitely fill the packet?

Reply:
Yeah, there were some pretty silly mistakes in the code :sweat_smile: